spec/unit/block_score/actions/delete_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'faker'
+
+RSpec.describe BlockScore::Actions::Delete do
+  describe '#delete' do
+    subject(:candidate) { create(:candidate) }
+    before { candidate.delete }
+
+    it { is_expected.not_to be_persisted }
+    it { expect(candidate.deleted).to be true }
```

Comment:
There is already a deleted? method. It checks the deleted flag. However the deleted flag (referenced here) is the public API.